lib/Cake/Model/Datasource/Database/Mysql.php

```diff
@@ -895,8 +895,10 @@ public function insertMulti($table, $fields, $values) {
 				$i++;
 			}
 		}
+		$this->_connection->query('SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE="NO_AUTO_VALUE_ON_ZERO"');
```

Comment:
Perhaps you can find a different way to do the 0 value insert other than in a fixture's default recordset.